src/main/groovy/com/avioconsulting/mule/MuleLinterCli.groovy

```diff
@@ -1,27 +1,25 @@
 package com.avioconsulting.mule
 
-import com.avioconsulting.mule.linter.model.Application
 import picocli.CommandLine
 
-@CommandLine.Command(name = "MuleLinter", header = "%n@|green Mule Linter|@")
+@CommandLine.Command(name = 'MuleLinter', header = '%n@|green Mule Linter|@')
 class MuleLinterCli implements Runnable {
 
-    @CommandLine.Option(names = ["-r", "--rules"], required = true, description = "Rule definition file")
+    @CommandLine.Option(names = ['-r', '--rules'], required = true, description = 'Rule definition file')
     String rules
 
-    @CommandLine.Option(names = ["-d", "--dir"], required = true, description = "Application Directory")
+    @CommandLine.Option(names = ['-d', '--dir'], required = true, description = 'Application Directory')
     String appDir
 
-
-    public static void main(String... args) {
+    static void main(String... args) {
         new CommandLine(new MuleLinterCli()).execute(args)
     }
 
     @Override
     void run() {
-        File appDir = new File(appDir)
-        Application app = new Application(appDir)
-        System.out.println(app.hasGitIgnore())
-        System.out.println(app.hasFile('test/file.txt'))
+        // Parse application to build out structure
+        MuleLinter ml = new MuleLinter(appDir)
+        ml.runLinter()
```

Comment:
Need to handle invalid app directory/not found more cleanly.  Currently an exception is just thrown:

adesjard@ac0050:~/code/mulesoft/avio/mule-linter/mule-linter/build/libs$ java -jar mule-linter-1.0-SNAPSHOT-all.jar -d /sdgfsfdgfds -r asdfsa
WARNING: An illegal reflective access operation has occurred
WARNING: Illegal reflective access by org.codehaus.groovy.reflection.ReflectionUtils (file:/home/adesjard/code/mulesoft/avio/mule-linter/mule-linter/build/libs/mule-linter-1.0-SNAPSHOT-all.jar) to constructor java.io.FileNotFoundException(java.lang.String,java.lang.String)
WARNING: Please consider reporting this to the maintainers of org.codehaus.groovy.reflection.ReflectionUtils
WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
WARNING: All illegal access operations will be denied in a future release
java.io.FileNotFoundException: /sdgfsfdgfds (Application directory does not exists.)
        at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
        at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
        at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
        at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:490)
        at org.codehaus.groovy.reflection.CachedConstructor.invoke(CachedConstructor.java:72)
        at org.codehaus.groovy.runtime.callsite.ConstructorSite$ConstructorSiteNoUnwrapNoCoerce.callConstructor(ConstructorSite.java:105)
        at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCallConstructor(CallSiteArray.java:59)
        at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:237)